Example/ChatExample.xcodeproj/project.pbxproj

```diff
@@ -507,7 +508,7 @@
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 456T4LX972;
```

Comment:
I should be able to set something so this doesn't happen but we should throw out the changes to the .xcodeproj